{{cookiecutter.project_slug}}/clients/web/vue3/src/views/RequestPasswordReset.vue

```diff
@@ -1,20 +1,43 @@
 <template>
-  <div class="request-password-reset">
-    <form @submit.prevent="attemptResetRequest()">
-      <InputField
-        v-model:value="form.email.value"
-        :errors="form.email.errors"
-        @blur="form.email.validate()"
-        type="email"
-        label="Email:"
-        placeholder="Email"
-      />
-
-      <button type="submit">Request Password Reset</button>
-    </form>
-  </div>
+  <template v-if="!resetLinkSent">
+    <div class="request-password-reset">
+      <form @submit.prevent="attemptResetRequest()">
+        <InputField
+          v-model:value="form.email.value"
+          :errors="form.email.errors"
+          @blur="form.email.validate()"
+          type="email"
+          label="Email:"
+          placeholder="Email"
+        />
+
+        <button
+          :class="{ 'btn--disabled': !form.isValid }"
+          :disabled="!form.isValid"
+          type="submit">
+          Request Password Reset
+        </button>
+      </form>
+    </div>
+  </template>
+  <template v-else>
```

Comment:
We should pull these out into constants for translation services in a near future